spring-rest2ts-generator/src/main/java/com/blueveery/springrest2ts/converters/JavaEnumToTsEnumConverter.java

```diff
@@ -35,7 +35,7 @@ public boolean preConverted(JavaPackageToTsModuleConverter javaPackageToTsModule
     public void convert(Class javaClass, NullableTypesStrategy nullableTypesStrategy) {
         TSEnum tsEnum = (TSEnum) TypeMapper.map(javaClass);
         for (Object enumConstant : javaClass.getEnumConstants()) {
-            tsEnum.getTsEnumConstantList().add(new TSEnumConstant(((Enum) enumConstant).name()));
+            tsEnum.add(((Enum) enumConstant).name());
```

Comment:
Method input arg ``javaClass`` has no parameterized class
You should consider changing signatures of methods 
``
    public abstract boolean preConverted(JavaPackageToTsModuleConverter javaPackageToTsModuleConverter, Class javaClass);
    public abstract void convert(Class javaClass, NullableTypesStrategy nullableTypesStrategy);
``
from ``ComplexTypeConverter.java`` by adding <?> to parametrized class.

Reply:
I tried to do it, but it requires large refactoring and testing so rejected :)

Reply:
moreover it is change of api used by users so it would require major release